robotoff/app/core.py

```diff
@@ -31,15 +31,14 @@ def get_insights(
     limit: Optional[int] = 25,
     offset: Optional[int] = None,
     count: bool = False,
-    latent: Optional[bool] = False,
 ) -> Iterable[ProductInsight]:
     if server_domain is None:
         server_domain = settings.OFF_SERVER_DOMAIN
 
-    where_clauses = [ProductInsight.server_domain == server_domain]
-
-    if latent is not None:
-        where_clauses.append(ProductInsight.latent == latent)
+    where_clauses = [
+        ProductInsight.server_domain == server_domain,
+        ProductInsight.latent == False,  # noqa: E712
```

Comment:
why use noqa and not use `not ProductInsight.latent` ? This is because it maybe None ?

Reply:
I used noqa because`ProductInsight.latent == False` is not PEP8 compliant.
This line is going to be deleted once the migration is done (we're going to delete the ProductInsight.latent field)

Reply:
I don't think that `not ProductInsight.latent` works in Peewee.
This line is going to be deleted once the migration is done.